values.go

```diff
@@ -327,6 +329,61 @@ func (n NullInt32) Encode(w *WriteBuf, oid Oid) error {
 	return encodeInt32(w, oid, n.Int32)
 }
 
+// Xid is PostgreSQL's Transaction ID type.
+//
+// In later versions of PostgreSQL, it is the type used for the backend_xid
+// and backend_xmin columns of the pg_stat_activity system view.
+//
+// Also, when one does
+//
+// 	select xmin, xmax, * from some_table;
+//
+// it is the data type of the xmin and xmax hidden system columns.
+//
+// It is currently implemented as an unsigned four byte integer.
+// Its definition can be found in src/include/postgres_ext.h as TransactionId
+// in the PostgreSQL sources.
+type Xid uint32
+
+// NullXid represents a Transaction ID (Xid) that may be null. NullXid implements the
+// Scanner and Encoder interfaces so it may be used both as an argument to
+// Query[Row] and a destination for Scan.
+//
+// If Valid is false then the value is NULL.
+type NullXid struct {
```

Comment:
Yes, definitely. Here's an example of me logged on to my personal copy of PostgreSQL 9.5, selecting both xid-typed columns from pg_stat_activity. The backend_xid column is null.

```
# \pset null '[NULL]'
# select backend_xid, backend_xmin from pg_stat_activity;
┌─────────────┬──────────────┐
│ backend_xid │ backend_xmin │
├─────────────┼──────────────┤
│      [NULL] │      1090671 │
└─────────────┴──────────────┘
(1 row)
```

One thing that prompted this PR was a desire to easily read every column from pg_stat_activity. This would be useful to a lot of PostgreSQL tool writers, as well as non-novice users looking to monitor the state connections to their database.